Packs/Okta/ReleaseNotes/2_1_7.md

```diff
@@ -0,0 +1,13 @@
+
+#### Classifiers
+##### Okta IAM - App Sync (classifier)
+- Added the classification for Okta log update user profile events to **IAM - App Update** incident type (used for the **Identity Lifecycle Management (ILM)** pack).
+
+#### Integrations
+##### Okta IAM
+- Added command: ***okta-iam-list-user-applications***.
```

Comment:
```suggestion
- Added the ***okta-iam-list-user-applications*** command.
```

Reply:
This is after rn review. Don't think we should change it